packages/fxa-settings/src/components/FormPasswordWithInlineCriteria/en.ftl

```diff
@@ -10,7 +10,7 @@ form-password-with-inline-criteria-reset-new-password =
   .label = New password
 form-password-with-inline-criteria-confirm-password =
   .label = Re-enter password
-form-password-with-inline-criteria-reset-submit-button = Reset password
+form-password-with-inline-criteria-reset-submit-button-2 = Create new password
```

Comment:
Since I haven't exposed this string for localization you can keep the old ID.

Reply:
I'll keep the updated id since the tests are already running. Thanks!